src/software/world/ball.cpp

```diff
@@ -64,6 +64,18 @@ BallState Ball::estimateFutureState(const Duration &duration_in_future) const
     return BallState(future_position, future_velocity);
 }
 
+bool Ball::hasBallBeenKicked(const Angle &expected_kick_direction,
+                             double min_kick_speed) const
+{
+    static constexpr Angle MAX_ANGLE_DIFFERENCE = Angle::fromDegrees(20);
```

Comment:
Nit:

Usually it's nice to have a comment on why a value was chosen (even if it's just noting that it is arbitrary)

Reply:
I added a clarifying comment. Would appreciate it if you could let me know if it can be clarified even further